userspace/libsinsp/filterchecks.cpp

```diff
@@ -2782,6 +2787,138 @@ uint8_t* sinsp_filter_check_thread::extract(sinsp_evt *evt, OUT uint32_t* len, b
 				RETURN_EXTRACT_STRING(m_tstr);
 			}
 		}
+		case TYPE_VPGID_NAME:
+		{
+			//
+			// Relying on the convention that a process group id is the process id of the process group leader
+			//
+			sinsp_threadinfo* vpgidinfo =
```

Comment:
uhm maybe I'm missing something but here we are accessing a table indexed by tid referred to the init namespace, with a tid referred to a possible different namespace :thinking:

Reply:
@Andreagit97 I investigated why it was working and you will see it in the last commit. On top I added a clearer distinction between host processes and pid namespace processes. Proposing to also address this for `m_sid` as maybe today we could potentially fetch the wrong information in some edge cases? Let me know and I can add that commit. Thanks.